<h1> My Jupyter Notebook on IBM Watson Studio.</h1>

<b> Navin Patwari </b><br>
Current Occupation : Student<br>
Desired Occupation : Data Scientist / Data Engineer

I am interested in data science beause I like playing with data and making predictions based on data sets</br> to improve users experience and at the same time reduce costs.
Also because of the wide applications of Data analysis it help me became a valuable asset to the society.

<h3> This code is supposed to display the sum of digits you enter</h3>

In [6]:
n = int(input())
d=0
while(n!=0):
    d+=n%10
    n//=10
print(d)

123
6


<hr>
Input the number you want to calculate sum of digit<br>
The program then does the following.</br>
<ol>
    <li>Modulo the number by 10</li>
    <li>Add the number to the variable 'd'</li>
    <li>display the result as 'd'</li>
</ol>
<hr>
<br>Sample Exeution on input = 123<br>
<table>
    <tr> 
        <th> d </th>
        <th> n </th>
    </tr>
    <tr>
        <th> 3 </th>
        <th> 12 </th>
    </tr>
    <tr>
        <th>  </th>
        <th> 1 </th>
        <th> 0 </th>
    </tr>
    

    